### ISSUE TITLE
how to import bvh

### ISSUE THREAD

**Issue by Dev5896** [carlosedubarreto/b3d_mocap_import#16]
thanks man for the help i am able to to bring in the mocap using smpl model in blender
I need to retarget it on make human model so i have to export it as a bvh file then retarget it can i avoid some of this steps to make the workflow faster ?

**Comment by carlosedubarreto** [comment]
I think that the biggest problem on the workflow is the lack of direct exporting metahuman for blender.
And I dont know any other way to bring to blender. I just know this method
https://www.youtube.com/watch?v=-UsZ6_c__uQ

And to be honest, I would have to watch this video again in case I need to use it again 😅
Only if someone creates a tool to do that, I guess...
Maybe Epic releases it on Quixel bridge to export to blender in the future...

**Comment by carlosedubarreto** [comment]
I was not sure about your question, I answered on what I thought was the problem 😊

**Comment by Dev5896** [comment]
can i import the mocap in bvh format directly without importing it through smpl model?

**Comment by carlosedubarreto** [comment]
SMPL is for the importing of json files from Easymocap only,

After importing the animation data you can export to BVH and smpl is not needed anymore.

**Comment by Dev5896** [comment]
yeah i am asking can i import the json files directly in bvh format?
is there any way to do that

**Comment by carlosedubarreto** [comment]
wich solution are you using?
Vibe, frankmocap,easymocap, mediapipe?

It depends on the solution you are using, it can be possible.

But I must say the best one I've found is the easymocap using the SMPL.
you can use the easymocp without the SMPL, but if much worst and dont have the finger movements.

**Comment by Dev5896** [comment]
I am asking for easymocap . ok i got it whatu are telling 
and is it possible to do motion capture in real time using any of the above vibe,frankmocap and easymocap?

**Comment by carlosedubarreto** [comment]
I've made a alpha version using mediapipe, but its very unreliable and crashy.
I fell that there is a lot of work to be done before going on the realtime road.

But you can try it.
Its not easy to use, you gonna have to know some python.
It can be downloaded from the gumroad or in this github( in this github, there is an alpha branch that you can download)

**Comment by Dev5896** [comment]
can u provide the link of the alpha version here and 
i have just started learning python  just finished the basics upto creating objects and class 
i am really looking to try that alpha version with a little bit guidance/help from u

**Comment by carlosedubarreto** [comment]
https://github.com/carlosedubarreto/b3d_mocap_import/tree/alpha_realtime

Or the alpha in this link
https://gumroad.com/l/mocap_import
